src/libraries/Microsoft.Extensions.Diagnostics.Configuration/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Extensions.Diagnostics
+
+`Microsoft.Extensions.Diagnostics` contains the default implementation of meter factory and extension methods for registering this default meter factory to the DI.
```

Comment:
Was this intended to go in M.E.D assembly instead of M.E.D.Configuration?

Reply:
Lots of copy paste docs that will get cleaned up once things are working.